etl/scripts/etl.py

```diff
@@ -22,6 +23,8 @@ def open_google_spreadsheet(docid):
     res = req.get(url)
     if res.ok:
         return BytesIO(res.content)
+    else:
+        print(f"Could not fetch {url}. Error code: {res.status_code}. Please check if the url is valid. The docid is hard coded in the python script.")
```

Comment:
Just curious, why not just print the docid? ;)

Reply:
Ease of use: I can copy paste this link or even click it in some editors.
And fits context of http error.